projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+# Reuse places & organizers
```

Comment:
```suggestion
# Reusing places & organizers
```

The UiTPAS guides always use this form for verbs in the names of guides, so I'd follow that convention here as well for consistency. (See for example https://docs.publiq.be/docs/uitpas/2d8ffb41e7f63-registering-ticket-sales)

You also already use this form in other headers on this page, for example "Storing the place ID(s) inside your application"